packages/electrode-archetype-react-component/config/webpack/partial/styles.js

```diff
@@ -10,6 +10,11 @@ const cssnext = archDevRequire("postcss-cssnext");
 const styleLoader = archDevRequire.resolve("style-loader");
 const cssLoader = archDevRequire.resolve("css-loader");
 const postcssLoader = archDevRequire.resolve("postcss-loader");
+const stylusLoader = archDevRequire.resolve("stylus-relative-loader");
+
+const configPath = Path.resolve("config.json");
```

Comment:
The direction we are going is to have an `archetype` directory that lets user put any archetype related config/customization there.